api/api.go

```diff
@@ -215,3 +219,10 @@ const (
 	Success CommandExecutionStatus = "SUCCESS"
 	Failure CommandExecutionStatus = "FAILURE"
 )
+
+type OutputType string
+
+const (
+	STRING OutputType = "STRING"
```

Comment:
Let's renamename these so it's more clear what it does
```
      OutputTypeString OutputType = "STRING"
      OutputTypeSecret OutputType = "SECRET"
```
so `api.OutputTypeString` is more clear